ansible/setup.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Configure node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    project_user: epoch
+    devops:
+      - dincho
+    developers:
+      - sennui
+      - velzevur
+
+  tasks:
+    - name: Create epoch daemon user
```

Comment:
I guess here you wanted to use `{{ project_user }}`. No need to change PR of course.

Reply:
Actually no, I meant the user for the daemon bin/epoch. Probably my english wording is not good enough?

Reply:
Actually no, I meant the user for the daemon bin/epoch. Probably my english wording is not good enough?

Reply:
No, it is ok. Thanks for explanation.